src/app/highlight-cards/HighlightCard.ts

```diff
@@ -105,3 +113,22 @@ function backgroundImage(sfApiHighlightCard: SfApiHighlightCard, donateUriPrefix
   return campaignFamilyBackgroundImages[sfApiHighlightCard.campaignFamily] || defaultBackground;
 }
 
+export function SFHighlightCardsToFEHighlightCards(apiHighlightCards: SfApiHighlightCard[]): HighlightCard[] {
+  function isChristmasChallenge(card: SfApiHighlightCard) {
+    return card.campaignFamily === "christmasChallenge";
+  }
+
+  // Array.prototype.sort is specified as being stable since (or ECMAScript 2019). I don't think we support any browsers
+  // too old to have that, and we can cope with the possibility of none CC cards being in the wrong order in very old
+  const cardsSortedCCFirst = apiHighlightCards.sort((a, b) => {
+    return +isChristmasChallenge(b) - +isChristmasChallenge(a);
+  });
```

Comment:
I just realised `sort` sorts in place, returning a ref to the original array not a new one. So it probably would be better to treat it as returning void for clarity. But doesn't make any difference to the result here and I think this is fine for now, we probably won't keep the code in this form long anyway.

Reply:
actually it is quite evil to do mutate an object passed in without advertising that you're going to declare it. I could easily use spread to clone the passed in array first. But in this case the mutation is not going to cause a problem so now that the PR is being reviewed and we'd presumably like to get it in prod quickly I might leave as is.  LMK if you think I should add the `const apiHighlightCardsCloned = [...apiHighlightCards];` @NoelLH

Reply:
I'm happy to leave it as we suspect it's temporary code @bdsl